js/main.js

```diff
@@ -20,24 +20,34 @@ function photoUpdate(event) {
 
 function submitForm(event) {
   event.preventDefault();
-  var formInput = {};
+
+  if (data.editing === null) {
+    var formInput = {};
+    formInput.nextEntryId = data.nextEntryId;
```

Comment:
Something here is weird. Each entry should have an `entryId`. `nextEntryId` is just part of the data model so that you know what the "next" entry id should be 😄